pyodide-build/pyodide_build/buildall.py

```diff
@@ -497,6 +498,138 @@ def generate_needs_build_set(
     return needs_build
 
 
+@dataclasses.dataclass(eq=False, repr=False)
```

Comment:
Why is `dataclass` used? It is not as a container for the data.

Reply:
It was originally. But yeah I'll remove it.